packages/block-editor/src/components/rich-text/index.js

```diff
@@ -39,6 +39,34 @@ import { useFormatTypes } from './use-format-types';
 import FormatEdit from './format-edit';
 import { getMultilineTag, getAllowedFormats } from './utils';
 
+/**
+ * Removes props used for the native version of RichText so that they are not
+ * passed to the DOM element and log warnings.
+ *
+ * @param {Object} props Props to filter.
+ *
+ * @return {Object} Filtered props.
+ */
+function removeNativeProps( props ) {
+	return omit( props, [
+		'__unstableMobileNoFocusOnMount',
+		'deleteEnter',
+		'placeholderTextColor',
+		'textAlign',
+		'selectionColor',
+		'tagsToEliminate',
+		'rootTagsToEliminate',
+		'disableEditingMenu',
+		'fontSize',
+		'fontFamily',
+		'fontWeight',
+		'fontStyle',
+		'minWidth',
+		'maxWidth',
```

Comment:
Is there a style equivalent for Native to avoid all these style props?

Reply:
I think in theory we could extract them from the style object that is passed to the native view 🤔 .